src/Autocomplete/assets/dist/controller.js

```diff
@@ -123,7 +126,6 @@ class default_1 extends Controller {
             this.element.innerHTML = currentHtml;
             this.initializeTomSelect();
             this.tomSelect.setValue(currentValue);
-            this.startMutationObserver();
```

Comment:
The `this.initializeTomSelect()` call 2 lines above already starts the mutation observer (only under certain conditions, E.G when not using remote data). So this call incorrectly starts the mutation observer in the cases when it shouldn't run, which is why I removed the call here.